test/xds/xds_server_serving_mode_test.go

```diff
@@ -386,7 +386,7 @@ func waitForFailedRPC(ctx context.Context, t *testing.T, cc *grpc.ClientConn) {
 		return
 	}
 
-	ticker := time.NewTimer(1 * time.Second)
+	ticker := time.NewTicker(1 * time.Second)
```

Comment:
One second is a long time to wait before making another RPC.

Also, this helper should ideally verify the status code and/or status error message as well.

Reply:
This was already is master, but I could perhaps change it. Switched the time to 10 milliseconds as per your other suggestion.